### DISCUSSION TITLE
Question: best solution for side-by-side WYSIWYG/Source editor setup

### DISCUSSION THREAD

**Discussion by larsw** [mdx-editor/editor#82] [Q&A]
Hi,

If I want to combine the mdx-editor in a side-by-side setup with an editor to handle the raw Markdown syntax (and keep the two in sync),
is CodeMirror the best editor framework to use? I know this is a YMMV situation, but I'm wondering if anyone already has some boileplate code for setting this up.

Thanks in advance.

**Comment by petyosi** [comment]
FWIW, I'm using codemirror for the source mode, but I'm not super happy with how it feels. I would say that there's a lot more that can be improved if you want to edit markdown as a source specifically. Of course, codemirror is a great general purpose framework, and there's a lot that can be done in it. 

If you feel like it, I'm happy to discuss if your setup can find its way as a contribution to the project.
